src/main/cpp/CMakeLists.txt

```diff
@@ -184,6 +184,7 @@ set(CUDFJNI_INCLUDE_DIRS
 
 add_library(
   spark_rapids_jni SHARED
+  src/Aggregation64Utils.cpp
```

Comment:
Let's use the name `AggregationUtils.*` so in the future we can put more aggregation-related stuff in.